mitodl/mitxpro
# Add enrollment to voucher on course enrollment

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #183 

#### What's this PR do?
On course enrollment, if the user has an active voucher for the course being enrolled in, attach the enrollment to the course.

#### How should this be manually tested?
Try enrolling for a course using a voucher and check that the voucher is now attached

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)

#### What GIF best describes this PR or how it makes you feel?
![](https://media.giphy.com/media/3owvJVUhntFJjT4usg/source.gif)
